pkg/kubectl/plugins/proxier.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugins
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/golang/glog"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/kubernetes/pkg/kubectl"
+)
+
+type proxier struct {
```

Comment:
was wondering if defining an interface for proxier makes sense here from unit testing perspective ?